pandas/tests/scalar/period/test_period.py

```diff
@@ -1038,9 +1038,10 @@ def test_add_raises(self):
             dt1 + dt2
 
     boxes = [lambda x: x, lambda x: pd.Series([x]), lambda x: pd.Index([x])]
```

Comment:
maybe move these 2 lines to the top of the file?

Reply:
If we use them more than once it makes sense, but ATM they are defined right next to where they are used, which is pretty ideal.